crates/pathfinder/src/rpc/types.rs

```diff
@@ -202,8 +202,6 @@ pub mod reply {
     pub enum BlockStatus {
```

Comment:
I guess [`Rejected`](https://github.com/starkware-libs/starknet-specs/blob/2cd3ec9f6183007b750c61c96f648bde7c1912ef/api/starknet_api_openrpc.json#L1422) is a dead status code in our case as we're not storing reorged blocks?

Reply:
Yes, it is. I thought we should just keep it so that we're aligned with the specs.